core/templates/dev/head/services/PlaythroughServiceSpec.js

```diff
@@ -16,104 +16,93 @@
  * @fileoverview Unit tests for the playthrough service.
  */
 
-describe('Playthrough service', function() {
+describe('PlaythroughService', function() {
```

Comment:
A general comment on the tests here... Do we need to guard against false positives? i.e. make sure that no action is recorded when the action doesn't actually happen.

Reply:
Nope, these functions are hooked up to where the actions happen so an e2e test is more appropriate (@pranavsid98 has already written one).